lib/samples/identify_features_in_wms_layer/identify_features_in_wms_layer.dart

```diff
@@ -0,0 +1,191 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import 'package:arcgis_maps_sdk/arcgis_maps.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/utils/sample_state_support.dart';
+import 'package:flutter/material.dart';
+import 'package:webview_flutter/webview_flutter.dart';
+
+class IdentifyFeaturesInWmsLayer extends StatefulWidget {
+  const IdentifyFeaturesInWmsLayer({super.key});
+
+  @override
+  State<IdentifyFeaturesInWmsLayer> createState() =>
+      _IdentifyFeaturesInWmsLayerState();
+}
+
+class _IdentifyFeaturesInWmsLayerState extends State<IdentifyFeaturesInWmsLayer>
+    with SampleStateSupport {
+  // Create a controller for the map view.
+  final _mapViewController = ArcGISMapView.createController();
+  // Create a WMS Layer.
+  late WmsLayer _wmsLayer;
+  // Create a web view controller for holding HTML content.
+  final _webViewController = WebViewController();
+  // A flag for when the map view is ready and controls can be used.
+  var _ready = false;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+        top: false,
+        child: Stack(
+          children: [
+            Column(
+              children: [
+                Expanded(
+                  // Add a map view to the widget tree and set a controller.
+                  child: ArcGISMapView(
+                    controllerProvider: () => _mapViewController,
+                    onMapViewReady: onMapViewReady,
+                    onTap: onTap,
+                  ),
+                ),
+              ],
+            ),
+            // Display a banner with instructions.
+            SafeArea(
+              child: IgnorePointer(
+                child: Container(
+                  padding: const EdgeInsets.all(10),
+                  color: Colors.white.withOpacity(0.7),
```

Comment:
download_preplanned_map_area.dart and identify_layer_features.dart have white-on-black. Should these banner styles be consistent one way or the other?

Reply:
I flipped it since the Basemap is dark as otherwise it's not very visible. So I figured either one is fine, since they're consistent just reversed.